dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Machines/ReactivePowerControlLoops/ReactivePowerControlLoop.mo

```diff
@@ -63,6 +63,12 @@ model ReactivePowerControlLoop "Simplified Reactive Power Control Loop model"
   parameter Types.ReactivePowerPu QStator0Pu "Start value of the generator stator reactive power in pu (base QNomAlt) (generator convention)";
   parameter Types.VoltageModulePu UStatorRef0Pu "Start value of the generator stator voltage reference in pu (base UNom)";
 
+// Limiter for UStatorRefPu
+  parameter Real UStatorRefMinPu "Minimum reference voltage for the generator voltage regulator in pu";
```

Comment:
For UStatorRefMinPu and UStatorRefMaxPu, could you use the Types.VoltageModulePu ?
Also, when we say "in pu", we usually try to give the pu base, here "in pu (base UNom)". Thanks